client/landing/stepper/declarative-flow/hundred-year-plan.ts

```diff
@@ -11,15 +10,16 @@ import {
 } from 'calypso/signup/storageUtils';
 import { SiteId, SiteSlug } from 'calypso/types';
 import { ONBOARD_STORE, USER_STORE } from '../stores';
-import { useLoginUrl } from '../utils/path';
+import { stepsWithRequiredLogin } from '../utils/steps-with-required-login';
 import { recordSubmitStep } from './internals/analytics/record-submit-step';
 import type { ProvidedDependencies, Flow } from './internals/types';
 
 const HundredYearPlanFlow: Flow = {
 	name: HUNDRED_YEAR_PLAN_FLOW,
 	get title() {
-		return translate( '100-year Plan' );
+		return ( getPlan( PLAN_100_YEARS )?.getTitle() || '' ) as string;
```

Comment:
This logic was here https://github.com/Automattic/wp-calypso/pull/92335/files#diff-300ce3543c1eefbdea56c0f0050d16dd9d3fc80e0acb1d107cdd9ce0f23abc79L97 and the result of this function is exactly `translate( '100-year Plan' );`